sympy/solvers/diophantine/diophantine.py

```diff
@@ -143,21 +143,17 @@ class DiophantineEquationType:
     Parameters
     ==========
 
-    equation
+    equation :
         The diophantine equation that is being solved.
-    free_symbols: list (optional)
+    free_symbols : list (optional)
         The symbols being solved for.
-
-    Attributes
-    ==========
-
-    total_degree
+    total_degree :
         The maximum of the degrees of all terms in the equation
-    homogeneous
+    homogeneous :
         Does the equation contain a term of degree 0
-    homogeneous_order
+    homogeneous_order :
         Does the equation contain any coefficient that is in the symbols being solved for
-    dimension
+    dimension :
```

Comment:
There are not parameters. These are class attributes, for example: https://numpydoc.readthedocs.io/en/latest/format.html#documenting-classes

Reply:
I don't think that we have to distinguish those for now.
Even if we can, I believe that it's more worse in the usage examples because both sections can be rendered differently or coming without field lists

Reply:
> I don't think that we have to distinguish those for now.

I think, that will be incorrect to list attributes in the parameters section.

> Even if we can, I believe that it's more worse in the usage examples because both sections can be rendered differently or coming without field lists

I don't understand, what you mean by that.

Reply:
![docs sympy org_latest_modules_geometry_lines html_highlight=class%20line](https://user-images.githubusercontent.com/34944973/98460824-e3a16080-21ea-11eb-8795-40f895eaa87e.png)

I don't think that our documentation builder builds attributes section correctly.
Here is an example that uses both Parameters and Assumptions, and I clearly see that Attributes section use different table, and also it's disorganized because Parameters sections are reordered regardless of how it is positioned in the docstring.

Reply:
These problems come from our outdated numpydoc builder, so I don't think that this need to be changed before switching to napoleon extension

Reply:
That's alright, we don't need to change before fixing the builder issues, but we just need to make sure that we don't list attributes in the parameters section and keep it the way it was before.

Reply:
Okay if it was the change with this new version, we may need to revert the specific problem

Reply:
I have changed it to attributes for now.